apply.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Apply</title>
+    <link rel="stylesheet" href="src/apply.css">
+    <link rel="stylesheet" href="src/main.css">
+</head>
+<div>
+<body>
+    <header>
+        <h1>
+            Catpplication Time!
+        </h1>
+    </header>
+
+    <main>
+        <form id="catpplication">
+            <section>
+                
+                <p>
```

Comment:
For this input you are wrapping the ```<label>``` and ```<input>``` in ```p``` tags but the rest of your inputs just have ```<label>``` and ```input``` tags that are associated by the ```for=``` property. I would consider keeping consistency throughout all files.